plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -422,6 +422,12 @@ function MosaicMenuItem:init()
     }
     self[1] = self._underline_container
 
+    if not Device:isTouchDevice() then
+        -- Make it bigger on non-touch devices, without affecting our sizing
+        -- we'll keep doing with that small self.underline_h
+        self._underline_container.linesize = 5
```

Comment:
@Frenzie will probably want us to use `Size.line.thin` above instead of my `self.underline_h = 1`, and `thick` here, or thicker (to add to size.lua) if you need it even thicker.

Reply:
That's correct. :+1:

Reply:
well if @Frenzie isn't going to bite my head off ;), I would add a `thickest = Screen:scaleBySize(5),` and use that one

I could add two sizes to size.lua:
```lua
thicker = Screen:scaleBySize(3),
thickest = Screen:scaleBySize(5),
```

https://github.com/koreader/koreader/blob/5affb68233fda994150df1da3d0b573402f20842/frontend/ui/size.lua#L63-L68

Reply:
I'm not sure if that value really fits within the design scheme. If you would like to put it in there because it'll be reused a name like `progress` might be better. If it's a one-off it can just stay here as is, just with the knowledge that it's generally to be avoided.

Reply:
No, anything like that would imply this is what you should choose if you're doing those things, which is not the case. Where else do you intend to use the value?

Reply:
Yes, something like non_touch_active could work, if reuse is intended.

Reply:
> Where else do you intend to use the value?

I don't. Just there for now. Perhaps it could be reused in the detailed list mode but that is all.